From 68a5b3036bb7d4b4b095a34d85a8873bcb5a442b Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Fri, 26 Apr 2013 16:54:23 +0200
Subject: [LIBREPORT PATCH 2/6] reporter-bugzilla: remove recently added aux.
 files from bz reports

last_occurrence - holds a time stamp of last problem occurrence
ureports_counter - holds a number of sent ureports

Closes #155

Signed-off-by: Jakub Filak <jfilak@redhat.com>
Signed-off-by: Martin Milata <mmilata@redhat.com>
---
 src/plugins/bugzilla_format.conf           | 2 +-
 src/plugins/bugzilla_format_libreport.conf | 2 +-
 src/plugins/bugzilla_formatdup.conf        | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/plugins/bugzilla_format.conf b/src/plugins/bugzilla_format.conf
index f668d17..cad5876 100644
--- a/src/plugins/bugzilla_format.conf
+++ b/src/plugins/bugzilla_format.conf
@@ -47,7 +47,7 @@ Additional info:: \
 		-component,-architecture,\
 	-analyzer,-count,-duphash,-uuid,-abrt_version,\
 	-username,-hostname,-os_release,-os_info,\
-	-time,-pid,-pwd,\
+	-time,-pid,-pwd,-last_occurrence,-ureports_counter,\
 	%reporter,\
 	%oneline
 
diff --git a/src/plugins/bugzilla_format_libreport.conf b/src/plugins/bugzilla_format_libreport.conf
index 40428a5..2ecdf6f 100644
--- a/src/plugins/bugzilla_format_libreport.conf
+++ b/src/plugins/bugzilla_format_libreport.conf
@@ -49,7 +49,7 @@ Additional info:: \
 		-component,-architecture,\
 	-analyzer,-count,-duphash,-uuid,-abrt_version,\
 	-username,-hostname,-os_release,-os_info,\
-	-time,-pid,-pwd,\
+	-time,-pid,-pwd,-last_occurrence,-ureports_counter,\
 	%reporter,\
 	%oneline
 
diff --git a/src/plugins/bugzilla_formatdup.conf b/src/plugins/bugzilla_formatdup.conf
index 99f63c2..f5e89f7 100644
--- a/src/plugins/bugzilla_formatdup.conf
+++ b/src/plugins/bugzilla_formatdup.conf
@@ -47,7 +47,7 @@
 		-component,-architecture,\
 	-analyzer,-count,-duphash,-uuid,-abrt_version,\
 	-username,-hostname,-os_release,-os_info,\
-	-time,-pid,-pwd,\
+	-time,-pid,-pwd,-last_occurrence,-ureports_counter,\
 	-var_log_messages,\
 	%reporter,\
 	%oneline
-- 
1.8.1.4

